docs/lob_guide/src/asciidoc/_chapters/work_with_lob.adoc

```diff
@@ -0,0 +1,738 @@
+////
+/**
+* @@@ START COPYRIGHT @@@
+*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+* @@@ END COPYRIGHT @@@
+*/
+////
+
+[#work with lob]
+= Work with LOB
+
+[#create a sql table with lob columns]
+== Create a SQL Table with LOB Columns
+
+When creating a SQL table with LOB columns, following relevant tables and files are created as well:
+
+* One LOB MD table.
+* Two dependent descriptor tables.
+* HDFS data file (locates at /user/trafodion/lobs) for each column.
+
+[#syntax]
+== Syntax
+
+```
+CREATE TABLE table-name (lob-column-spec[, lob-column-spec]…)
+```
+
+```
+lob-column-spec is:
+column {lob type}
+
+lob type is:
+BLOB | CLOB [({numeric literal} [unit])] [STORAGE 'storage literal']
+
+unit is:
+empty | 
+K     | 
+M     | 
+G 
+```
+
+[#semantics]
+=== Semantics
+

```

Comment:
What happens if the STORAGE clause is not present? Is there any limit on LOB size then?

Reply:
1. STORAGE clause 

Syntax:
...
lob-type is:
BLOB | CLOB [({numeric literal} [unit])] [STORAGE 'storage literal']
...

So the STORAGE clause is optional, it's ok if you don't specify it (the attachment
![create a sql table with lob columns](https://user-images.githubusercontent.com/20532956/30423142-8cfd079a-9974-11e7-9eb9-151183271ee5.jpg)
).

2.  limit on LOB size
I've asked Sandhya the some question, the answer from her:

Minimum size:
We allow for an “empty” LOB so technically that’s a  0 length lob. 
We can insert using empty_blob(), empty_clob()  syntax.

Maxmum size:
Currently this is set to 10GB and it's controlled by the CQD LOB_MAX_SIZE. 
But this can be changed depending on the customers needs to whatever they want. 
There isn’t a real limit as such so we could go to 8 terabytes if needed but the user will have to configure enough HDFS disks to be able to support such a large LOB and several of them potentially.